libs/commons/List_.ml

```diff
@@ -222,6 +222,12 @@ let tl_exn errmsg xs =
   | [] -> failwith errmsg
```

Comment:
Despite you checked the box I see no changelog.

Reply:
This diff alone doesn't change any user facing behavior, so I wasn't sure that it required, but I will add one.